### DISCUSSION TITLE
Reload saved preset

### DISCUSSION THREAD

**Discussion by mattias221** [apohl79/audiogridder#1102] [Help]
Hi

I' so glad for this solution of audiogridder. Now I can run my favourite windows software synth on my MacBook. 🙏

I'm having a reload project issue.
When loading a saved project in my daw (Logic) the software synth doesn't load the preset that I have chosen and saved in the project.

![image](https://user-images.githubusercontent.com/119296093/204252565-81465b56-6094-4fc1-964e-68a7135985cb.png)

The enclosed video replicate what I have described.

https://drive.google.com/file/d/1zZkRfNfWWqSUnk5FaVUaSedxaCi4cvT8/view

Within the opened project the selected preset stays, but not when closing and opening the project again.

Any suggestions? 🙂

Mattias

**Comment by apohl79** [comment]
Could you share the plugin logfile? When you hit save in logic, AG will request the state of each loaded plugin and persist it in the logic session (and restores it when loading the session later). Not sure why it does not work in your case.

**Comment by mattias221** [comment]
Thank you very much!
Where can I find the plugin logfile?
What is kind of peculiar is that Logic recalls some of the knob settings of
the synth. But not everyone. E.g. if the wheel knob is set to top or bottom
it remembers but not if the saw is on or of.
It would be really helpful being able to save and recall settings.

Den ons 7 dec. 2022 kl 11:50 skrev Andreas Pohl ***@***.***>:

**Reply by apohl79** [reply]
https://audiogridder.com/bug-report-diagnostics-locations/

I assume that the synth is behaving correctly when loaded into logic or another daw directly?

**Comment by mattias221** [comment]
Yes it is. I can confirm from my pc laptop.

torsdag 8 december 2022 skrev Andreas Pohl ***@***.***>:

**Reply by apohl79** [reply]
Ok, could you try one last thing. In the plugin, change the "Remote Sync Frequency" to every 10 seconds. After loading your preset, wait 10s before saving and closing the session. Now see if the synth reloads with the final settings.

**Comment by mattias221** [comment]
Thank you very much. I will try that and then write a line here with the
results.

fredag 9 december 2022 skrev Andreas Pohl ***@***.***>:

**Comment by mattias221** [comment]
Thank you for your suggestions. Unfortunately changing the remote sync did
not help.
I guess I have to get a long with reopening the presets when opening a
project.
For now that is working.

An other technical wlan question.

I have used audiogridder on wifi.
I like to get rid of pops and short silences.
After reading on the forum I have understood that I need a physical router.
And as I have understood it needs a WAN port ? I don't understand how that
works or what I need.

For now, I only have a small WLAN-router (4G/LTE -
https://www.amazon.se/TP-Link-M7200-WLAN-router-nedladdning-uppladdning/dp/B079GZNQ2B?th=1
)
I like to use audiogrid only on my Macbook Pro which only have 4 USB-C
ports.

What to I need to buy?

(I have also post this second question in a new issue:
https://github.com/apohl79/audiogridder/issues/1118. I clean up when I get
an answer. 🙂)

Regards
Mattias


Den tis 13 dec. 2022 kl 13:51 skrev Mattias Rosén ***@***.***>:
